### PR TITLE
Implementation of track-germs

### PR DESCRIPTION
**Pull request by YiskaPanksi**
I did the work, please review

### CODE REVIEW

File: 1 tables.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+create database TrackGermDB
+go
+use TrackGermDB
+go
+create table dbo.germ(
```
Comment (mordechaigellis) [inline 1 tables.sql:5]: missing pk

File: 2 sample data.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+select 'Norovirus', 'human'
```
Comment (mordechaigellis) [inline 2 sample data.sql:2]: missing insert

**Review by mordechaigellis — CHANGES_REQUESTED** [review]
missing basic sql

File: 1 tables.sql (hunk 1 above)
Comment (YiskaPanksi) [reply 1 tables.sql:5]: PK not needed....

File: 2 sample data.sql (hunk 2 above)
Comment (YiskaPanksi) [reply 2 sample data.sql:2]: Will fix

**Review by YiskaPanksi — COMMENTED** [review]
Agreed on one and disagreed on other....

File: 1 tables.sql (hunk 1 above)
Comment (mordechaigellis) [reply 1 tables.sql:5]: it is needed please do it

File: 2 sample data.sql (hunk 2 above)
Comment (YiskaPanksi) [reply 2 sample data.sql:2]: Fixed

File: 1 tables.sql (hunk 1 above)
Comment (YiskaPanksi) [reply 1 tables.sql:5]: Fixed

File: 2 sample data.sql (hunk 2 above)
Comment (mordechaigellis) [reply 2 sample data.sql:2]: all good

File: 1 tables.sql (hunk 1 above)
Comment (mordechaigellis) [reply 1 tables.sql:5]: all good

**Review by mordechaigellis — APPROVED** [review]
all is reviewed and good

**Comment by mordechaigellis** [comment]
Mark :85% deduct points because we had several attempts...

**Pull request closed without merging**